### ISSUE TITLE
Problem with dynamicaly created content

### ISSUE THREAD

**Issue by ivandotv** [luis-almeida/jPages#19]
I'm loading image thumbs from flickr. 
When all the info from flickr is loaded (callback response),  unordered list is created, after that jpages plugin is instantiated.
But it seems that the plugin is not aware of the list elements (children - thumbs) inside the container.
It fires the callback, but there are no items in the items object, although if i query the container element directly inside the callback via jquery it gives me all the children of the container.
The workaround that I'm currently using is setTimeout (fired 1000ms after flickr callback)
If I delay running the plugin , then it works.

Also, strange thing  it works when loading from cache.

**Comment by dcolumbus** [comment]
Yes, I'm trying to update the plugin dynamically as well and even though 62 items are loaded, jPages only seems to think there's 13.

What's the proper way to dynamically update jPages?
